docs/ember/components.md

```diff
@@ -102,75 +127,6 @@ export default class ArgsDisplay extends Component {
 </ul>
 ```
 
-### Understanding `args`
```

Comment:
Let's go ahead and leave this section (I'll do a follow-up on it to clean it up further). If you can just drop it back in, the rest of it looks like a great first iteration on this update, and will already be a big improvement for users. Thank you!

Reply:
I've added that section back in. I also started to update the section but got out of my depth pretty quickly so will leave the rest with you 🤣